recipes/benchmarks/fmbench/README.md

```diff
@@ -0,0 +1,117 @@
+|---	|---	|
+|experiment_name	|llama2-13b-inf2.24xlarge	|
+|payload_file	|payload_en_3000-3840.jsonl	|
+|instance_type	|ml.inf2.24xlarge	|
+|concurrency	|**	|
+|error_rate	|**	|
+|prompt_token_count_mean	|3394	|
+|prompt_token_throughput	|2400	|
+|completion_token_count_mean	|31	|
+|completion_token_throughput	|15	|
+|latency_mean	|**	|
+|latency_p50	|**	|
+|latency_p95	|**	|
+|latency_p99	|**	|
+|transactions_per_minute	|**	|
+|price_per_txn	|**	|
+
+<sup>1</sup> ** represent values hidden on purpose, these are available when you run the tool yourself.
+
+The report also includes latency Vs prompt size charts for different concurrency levels. As expected, inference latency increases as prompt size increases but what is interesting to note is that the increase is much more at higher concurrency levels (and this behavior varies with instance types).
+
+![Effect of prompt size on inference latency for different concurrency levels](./img/latency_vs_tokens.png)
+
+### How to get started with `FMBench`
+
+The following steps provide a Quick start guide for `FMBench`. For a more detailed DIY version, please see the [`FMBench Readme`](https://github.com/aws-samples/foundation-model-benchmarking-tool?tab=readme-ov-file#the-diy-version-with-gory-details).
+
+1. Launch the AWS CloudFormation template included in this repository using one of the buttons from the table below. The CloudFormation template creates the following resources within your AWS account: Amazon S3 buckets, Amazon IAM role and an Amazon SageMaker Notebook with this repository cloned. A read S3 bucket is created which contains all the files (configuration files, datasets) required to run `FMBench` and a write S3 bucket is created which will hold the metrics and reports generated by `FMBench`. The CloudFormation stack takes about 5-minutes to create.
+
+   |AWS Region                |     Link        |
+   |:------------------------:|:-----------:|
+   |us-east-1 (N. Virginia)    | [<img src="./img/CFT.png">](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=fmbench&templateURL=https://aws-blogs-artifacts-public.s3.amazonaws.com/artifacts/ML-FMBT/template.yml) |
+   |us-west-2 (Oregon)    | [<img src="./img/CFT.png">](https://console.aws.amazon.com/cloudformation/home?region=us-west-2#/stacks/new?stackName=fmbench&templateURL=https://aws-blogs-artifacts-public.s3.amazonaws.com/artifacts/ML-FMBT/template.yml) |
+
+1. Once the CloudFormation stack is created, navigate to SageMaker Notebooks and open the `fmbench-notebook`.
+
+1. On the `fmbench-notebook` open a Terminal and run the following commands.
+
+    ```{.bash}
+    conda create --name fmbench_python311 -y python=3.11 ipykernel
+    source activate fmbench_python311;
+    pip install -U fmbench
+    ```
+
+1. Now you are ready to `fmbench` with the following command line. We will use a sample config file placed in the S3 bucket by the CloudFormation stack for a quick first run.
```

Comment:
numbered list update

Reply:
also it has format issues.

Reply:
![image](https://github.com/meta-llama/llama-recipes/assets/14024481/7b2e9989-fad0-4c4e-a1db-3be18709e1e7)
Sorry could not follow, the attached image is what I see.